src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigStorageLocation.cs

```diff
@@ -30,9 +33,25 @@ private static Optional<bool> ParseBool(string str)
         private static readonly Func<string, Optional<int>> s_parseInt32 = ParseInt32;
         private static Optional<int> ParseInt32(string str)
             => int.TryParse(str, out var result) ? result : new Optional<int>();
+
+        private static readonly Func<string, Optional<string>> s_parseString = ParseString;
+        private static Optional<string> ParseString(string str)
+        {
+            if (str.Equals("unset", StringComparison.Ordinal))
```

Comment:
on interesting.  is this documented somewhere?

Reply:
https://editorconfig-specification.readthedocs.io/en/latest/#supported-pairs